src/test/java/com/ayprise/inventory/model/PurchaseTests.java

```diff
@@ -0,0 +1,154 @@
+package com.ayprise.inventory.model;
+
+import java.util.List;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class PurchaseTests {
+    // Product constants
+    Product switch2Product = Product.builder()
+            .withName("Nintendo Switch 2 Mario Kart Bundle")
+            .withCategory("Electronics")
+            .withDescription("Nintendo Switch 2 Mario Kart Bundle")
+            .build();
+    Product airpodsProduct = Product.builder()
+            .withName("Airpods Pro 2")
+            .withCategory("Electronics")
+            .withDescription("Apple Headphones")
+            .build();
+
+
+    @Test
+    @DisplayName("Create a purchase and test the total cost")
+    void getTotalTest() {
+        PurchaseItem switch2Purchase1 = PurchaseItem.builder()
```

Comment:
You should use `final var` instead.

```suggestion
        final var switch2Purchase1 = PurchaseItem.builder()
```